Traceback (most recent call last):
  File "/Users/filip/thesis/Classifier.py", line 534, in <module>
    fire.Fire(Handler)
  File "/Users/filip/opt/anaconda3/lib/python3.9/site-packages/fire/core.py", line 141, in Fire
    component_trace = _Fire(component, args, parsed_flag_args, context, name)
  File "/Users/filip/opt/anaconda3/lib/python3.9/site-packages/fire/core.py", line 475, in _Fire
    component, remaining_args = _CallAndUpdateTrace(
  File "/Users/filip/opt/anaconda3/lib/python3.9/site-packages/fire/core.py", line 691, in _CallAndUpdateTrace
    component = fn(*varargs, **kwargs)
  File "/Users/filip/thesis/Classifier.py", line 505, in __init__
    Handler.run(self.classifier, self.pool_type)
AttributeError: type object 'Handler' has no attribute 'handler'